schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -658,6 +658,9 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-04-01/Microsoft.Compute.json#/resourceDefinitions/disks" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-04-01/Microsoft.Compute.json#/resourceDefinitions/snapshots" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-04-01/Microsoft.ContainerInstance.json#/resourceDefinitions/containerGroups" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Compute.Galleries.json#/resourceDefinitions/galleries" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Compute.Galleries.json#/resourceDefinitions/galleries_images" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Compute.Galleries.json#/resourceDefinitions/galleries_images_versions" },                  
```

Comment:
>                   [](start = 167, length = 18)

remove the extra spaces.

Reply:
Do you mean the trailing spaces on line 663? If so, I have removed.